modules/browser/BrowserCapabilities.js

```diff
@@ -241,11 +241,32 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean}
      */
     usesNewGumFlow() {
-        return (this.isChrome()
-                && !this.isVersionLessThan('61'))
-            || this.isFirefox()
-            || this.isSafariWithWebrtc();
+        const REQUIRED_VERSION = 61;
+
+        if ((this.isChrome() && !this.isVersionLessThan(REQUIRED_VERSION))
+                || this.isFirefox() || this.isSafariWithWebrtc()) {
+            return true;
+        }
+
+        if (this.isNWJS()) {
+            // eslint-disable-next-line no-undef
+            const version = Number.parseInt(process.versions.chromium, 10);
+
+            return version >= REQUIRED_VERSION;
+        }
+
+        if (this.isChromeLike()) {
+            const ua = navigator.userAgent;
+
+            if (ua.match(/Chrome/)) {
```

Comment:
this also seems like overalp with https://github.com/jitsi/js-utils/pull/6 ?

Reply:
so we detect chrome-like things as chrome, but don't parse the chrome version but instead the actual product (electron, etc.) version?  this seems like it would get confusing/misleading, as we represent things as chrome but parse a version that isn't actually the chrome version?

Reply:
No, that's not what happens. Sort of. When `isChrome` returns true, that browser is detected as Chrome, so the version is the Chrome version. This is the case for Chrome, Chromum, Brave and after my js-utils patch, Vivaldi.

Then we have Electron, NWJS and Opera for which `isChrome` returns false, and their version is the product version. These are "Chrome like" and we want to extract the Chrome version by hand. I'll add a comment which hopefully clarifies this.

Thanks for being so thorough!